resources/views/vendors/vendors/forms/addVendorModal.blade.php

```diff
@@ -0,0 +1,68 @@
+<div class="modal-body h6">				
+    <div class="form-group row">
+        <label for="name" class="col-md-4 col-lg-2 col-form-label">Vendor Name:<span class="text-danger ml-1">*</span></label>
+        <input type="text" name="name" class="form-control col-lg-4">
+        <label for="email" class="col-md-4 col-lg-2 col-form-label">Address:</label>
+        <input type="text" name="address" class="form-control col-lg-4">
+    </div>
+    <div class="row form-group">
+        <label for="email" class="col-md-4 col-lg-2 col-form-label">City:</label>
+        <input type="text" name="city" class="form-control col-lg-4">
+        <label for="email" class="col-md-4 col-lg-2 col-form-label">Country:</label>
+        <input type="text" name="country" class="form-control col-lg-4">
+    </div>
+    <div class="form-group row">
+        <label for="email" class="col-md-4 col-lg-2 col-form-label">TIN number:<span class="text-danger ml-1">**</span></label>
+        <input type="text" name="tinNum" class="form-control col-lg-4">
+        <label for="email" class="col-md-4 col-lg-2 col-form-label">Email:</label>
+        <input type="email" name="email" class="form-control col-lg-4">
+    </div>
+    <div class="row form-group">
+        <label for="email" class="col-md-4 col-lg-2 col-form-label">Phone number 1:</label>
+        <input type="text" name="phone1" class="form-control col-lg-4">
+        <label for="email" class="col-md-4 col-lg-2 col-form-label">Phone number 2:</label>
+        <input type="text" name="phone1" class="form-control col-lg-4">
+    </div>
+    <div class="row form-group">
+        <label for="email" class="col-md-4 col-lg-2 col-form-label">FAX:</label>
+        <input type="text" name="fax" class="form-control col-lg-4">
+        <label for="email" class="col-md-4 col-lg-2 col-form-label">Mobile number:</label>
+        <input type="text" name="mobile_num" class="form-control col-lg-4">
+    </div>
+    <div class="row form-group">
+        <label for="email" class="col-md-4 col-lg-2 col-form-label">Contact Person:</label>
+        <input type="text" name="contact_person" class="form-control col-lg-4">
```

Comment:
Sir @justinmanigo , what field is linked to `Contact Person`? The `mobile number` field? I think this one needs to be grouped to minimize confusion.